guides/common/Makefile

```diff
@@ -10,7 +10,8 @@ DEST_HTML = $(DEST_DIR)/index-$(BUILD).html
 DEST_PDF = $(BUILD_DIR)/$(subst doc-,,$(NAME))-$(BUILD).pdf
 IMAGES_DIR = $(DEST_DIR)/images
 IMAGES_TS = $(DEST_DIR)/.timestamp-images
-CSS_SOURCES = $(shell find $(COMMON_DIR)/sass -type f)
+CSS_SOURCES = $(shell find $(COMMON_DIR)/css -type f)
+DOCINFO_SOURCES = $(shell find $(COMMON_DIR)/ -type f -name docinfo\*.html)
```

Comment:
Running `find . -type f -iname "docinfo.html"` in the repo returns exactly one file: `./guides/common/docinfo.html`. Why the `*`?

Reply:
There can be multiple, for header, footer etc, see asciidoctor docs. This is just more generic, I planned to use multiple but landed on a single one.

Reply:
OK, then we basically would not need a generic solution right now. However, I have no strong feelings. Your call.